### DISCUSSION TITLE
Should moderation be enabled by default?

### DISCUSSION THREAD

**Discussion by acheong08** [acheong08/ChatGPT#244] [Polls]
Refer to https://github.com/acheong08/ChatGPT/issues/245

**Comment by alexandreteles** [comment]
Maybe provide moderation as an option that can be passed to the Chatbot class?

**Reply by acheong08** [reply]
Possible. I'll implement this tomorrow. Should it be enabled or disabled by default though?

**Reply by zwa73** [reply]
> Possible. I'll implement this tomorrow. Should it be enabled or disabled by default though?

I don't think it should be on by default
For a new, and not exactly good, feature, it shouldn't affect the previous effect by default
And, I think everyone likes the current effect

**Comment by haok1402** [comment]
Is the moderation supported in V1?
